docs/skills/udev-rules.md

```
@@ -0,0 +1,99 @@
+
+# udev Rules — Prefix Taxonomy and Conventions
+
+## Prefix Allocation
+
+| Range | Purpose | Examples |
+|---|---|---|
+| `10–19` | Generic catch-all / hard-to-classify | `10-switch.rules` |
+| `50–59` | OEM/hardware quirks (device-specific patches) | `50-framework16.rules`, `50-zsa.rules`, `50-steam-horipad-controller.rules` |
+| `60–69` | Platform workarounds (kernel/driver patches) | `60-amd-s2idle-fixes.rules`, `60-arduino-mbed.rules` |
+| `70–79` | Security / authentication devices | `70-titan-key.rules`, `70-wooting.rules`, `70-u2f.rules` (build-injected) |
+| `71–` | **Reserved for build-injected game-devices-udev** — do not use statically |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)